CHANGES.md

```diff
@@ -1,5 +1,6 @@
 # 3.0.0 UNRELEASED
 - [FIXED] Expose `BasePlugin` type in Cloudant client.
+- [REMOVED] Remove support for virtual hosts.
```

Comment:
Let's expand on this a little
`Removed support for the deprecated Cloudant virtual hosts feature.`